src/Compilers/Core/Portable/PEWriter/CustomDebugInfoWriter.cs

```diff
@@ -247,19 +249,18 @@ private static void SerializeDynamicLocalInfo(IMethodBody methodBody, ArrayBuild
             cmw.WriteUInt32(4 + 4 + 4 + (uint)dynamicLocals.Count * blobSize);//Size of the Dynamic Block
             cmw.WriteUInt32((uint)dynamicLocals.Count);
 
-            int localIndex = 0;
             foreach (ILocalDefinition local in dynamicLocals)
             {
-                if (local.Name.Length > 63)//Ignore and push empty information
+                if (local.Name.Length >= identifierSize)//Ignore and push empty information
```

Comment:
Why not ```> identifierSize```?  Do the identifier names in the attribute have a termination character?